spec/system/admin/adding_to_an_order_spec.rb

```diff
@@ -287,9 +288,12 @@
         expect(page).not_to have_content("Cross Core Project ID")
         expect(page.has_selector?("option", text: cross_core_product_facility.name, visible: false)).to be(true)
         expect(page.has_selector?("option", text: product.name, visible: false)).to be(true)
+        expect(page.has_selector?("option", text: other_account.to_s, visible: false)).to be(false)
 
         select_from_chosen facility2.name, from: "add_to_order_form[facility_id]"
         select_from_chosen cross_core_product_facility2.name, from: "add_to_order_form[product_id]"
+        # This fails because the account is not valid for the facility
+        # select_from_chosen other_account.to_s, from: "Payment Source"
```

Comment:
For some reason `other_account` is not available for the facility, looks like a setup issue

Reply:
Fixed!